docs/manual/developer/02_building_complianceascode.md

```diff
@@ -136,23 +153,41 @@ make -j4 rhel7-profile-playbooks # Ansible Playbooks for all RHEL7 profiles
 make -j4 rhel7  # everything above for RHEL7
 ```
 
-(optional) Configure options before building using a GUI tool:
+### Configuring CMAKE options using GUI
+
+Configure options before building using a GUI tool:
 
 ```bash
 cd build/
 cmake-gui ../
 make -j4
 ```
 
-(optional) Use the `ninja` build system (requires the `ninja-build` package):
+### Reproducible Builds
+
+Set the environment variable `SOURCE_DATE_EPOCH` to generate [reproducible builds](https://reproducible-builds.org/).
+For details about the values and meaning of this variable please check this [source](https://reproducible-builds.org/specs/source-date-epoch/):
+
+```bash
+cd build/
+SOURCE_DATE_EPOCH=1614699939 make -j4
```

Comment:
Are there any implications of using parallel builds on the reproducibility?

Reply:
I don't think so. It is just a copy and paste matter. I'll remove it.